pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -174,9 +172,16 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, nil
 	}
 
-	orgParentFolderID, err := util.GetGCPParentFolderFromConfigMap(r.client, operatorNamespace, orgGcpConfigMap)
+	// Get operatorConfigMap with values, use it for getting some value
+	// If any field does not filled by GetOperatorConfigMap, check which mapping is empty and return err
```

Comment:
In my opinion, you don't need these comments, your code is very well speaking for itself (despite the function name `CheckValueNotExist`). :)

Reply:
haha I'll remove them